src/Selenium2Driver.php

```diff
@@ -773,6 +776,14 @@ private function clickOnElement(Element $element): void
         $element->click();
     }
 
+    private function scrollIntoView(Element $element): void
+    {
+        $this->executeJsOnElement(
+            $element,
+            "arguments[0].scrollIntoView({ behavior: 'instant', block: 'end', inline: 'nearest' });"
```

Comment:
shouldn't we use `nearest` as well in the `block` direction ?

Reply:
I don't use the `scrollIntoView` method myself. I've only copied the whole line from the PHP WebDriver (see https://github.com/php-webdriver/php-webdriver/blob/11923b4ba312fc9d137266ae67438be65d4d500f/lib/Remote/RemoteWebElement.php#L249 ). I trust that they know what's right here.